xml/gl.xml

```diff
@@ -6782,17 +6801,24 @@ typedef unsigned int GLhandleARB;
         <enum value="0x9535" name="GL_SUBGROUP_QUAD_ALL_STAGES_KHR"/>
         <enum value="0x9536" name="GL_MAX_MESH_TOTAL_MEMORY_SIZE_NV"/>
         <enum value="0x9537" name="GL_MAX_TASK_TOTAL_MEMORY_SIZE_NV"/>
+        <enum value="0x9538" name="GL_MAX_MESH_OUTPUT_VERTICES_EXT"/>
```

Comment:
The following GL enums should not be aliased with the NV extension because they would report different values:
- GL_MAX_MESH_OUTPUT_PRIMITIVES_EXT
- GL_MAX_MESH_WORK_GROUP_INVOCATIONS_EXT
- GL_MAX_MESH_WORK_GROUP_SIZE_EXT
- GL_MAX_TASK_WORK_GROUP_INVOCATIONS_EXT
- GL_MAX_TASK_WORK_GROUP_SIZE_EXT

(Compare the values reported in `VK_NV_mesh_shader` and `VK_EXT_mesh_shader` for reference.)

Reply:
Is this just implementation difference? The spec (either OpenGL or Vulkan) has same description of these enums (or same thing but different words).

Reply:
Is it just NVIDIA driver coding miss-alignment or NVIDIA driver do need to implement different values? As I'm wondering why same parameters have different values between NV and EXT extensions.

Reply:
We need to advertise different values for "NV" and "EXT" variants of the extension, so the enum values can't alias.